dev/requirements.txt

```diff
@@ -1,2 +1,3 @@
-libmambapy>=0.23
+conda-forge::libmambapy>=0.23
 conda-forge::libcurl
+conda-forge::libarchive
```

Comment:
I tried but it won't solve :(

Reply:
It does solve but we have the libarchive SONAME incompatibility:

https://github.com/conda-incubator/conda-libmamba-solver/runs/6797972107?check_suite_focus=true#step:4:351

Reply:
It seems to work now if we don't force libcurl from conda-forge (we had to before because it was the only build with unicode on windows, but defaults has that too now!). So I'll merge once everything is passing